dacapo/utils/voi.py

```diff
@@ -10,6 +10,7 @@
 import scipy.sparse as sparse
 
 
+# TODO: Why are we not using this?
```

Comment:
you mean for loss calculation or for plotting ?

Reply:
this is a dumb comment and should be removed. lol